### DISCUSSION TITLE
Shared hidden album ends up in the non-hidden folder of the coloaborator/observer

### DISCUSSION THREAD

**Discussion by pchampio** [ente-io/ente#4411] [Enhancements]
Shared hidden album ends up in the non-hidden folder of the collaborator/observer.
I would prefer that the collaborator(s)/observer(s) of a hidden albums that I share also is located in the hidden folder of the   collaborator(s)/observer(s).

A similar feature would be to be able to put an album that has been shared with you in your hidden folder. Regardless if it was hidden by the source or not.
The above feature would be to set the shared hidden album in the collaborator(s)/observer(s) hidden folder, with the ability to un-'hidden' it. 
great software btw :heart:

**Comment by huntfx** [comment]
I looked up this exact problem a few days back and found there's an existing request for it - #1571

**Comment by pchampio** [comment]
Duplicate of https://github.com/ente-io/ente/discussions/1571
